src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private String name = "Circle";
+    private int radius;
+
+    public Circle(Colors color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public double getRadius() {
+        return radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * getRadius() * getRadius();
```

Comment:
ми можемо в середині класу напряму звертатись до полів цього класу
```suggestion
        return Math.PI * radius * radius;
```